packages/core/src/browser/style/variables-bright.useable.css

```diff
@@ -33,10 +33,12 @@ is not optimized for dense, information rich UIs.
 
   --theia-ui-font-scale-factor: 1.2;
   --theia-ui-font-size0: calc(var(--theia-ui-font-size1) / var(--theia-ui-font-scale-factor));
-  --theia-ui-font-size1: 13px; /* Base font size */
```

Comment:
I believe we should revert this change, the comment is directed specifically for the variable:
```css
--theia-ui-font-size1: 13px; /* Base font size */
```